.github/workflows/lint.yaml

```diff
@@ -194,3 +194,63 @@ jobs:
             libusb-1.0-0-dev liblo-dev libavahi-client-dev python3-numpy
       - name: flake8
         run: make flake8 VERBOSE=1
+  spellintian:
+    name: spellintian
+    runs-on: ubuntu-latest
+    container: debian:stable
+    needs: build
+    steps:
+      - name: Download built source tree archive
+        uses: actions/download-artifact@v3
+        with:
+          name: ola-debian-stable-built-source-tree
+          path: .
+      - name: SHA256 artifact archive
+        run: sha256sum ola-debian-stable-built-source-tree.tar.gz
+      - name: Unarchive artifacts and delete archive
+        shell: bash
+        run: |
+          tar -xvzf ola-debian-stable-built-source-tree.tar.gz .
+          rm ola-debian-stable-built-source-tree.tar.gz
+      - name: Display structure of extracted files
+        if: env.ACTIONS_STEP_DEBUG == 'true'
+        run: ls -alR
+      - name: Update package database
+        run: apt-get update -y
+      - name: Install lint tools
+        run: apt-get -y install moreutils lintian
+      - name: Enable Problem Matcher for GitHub annotations
+        run: echo "::add-matcher::.github/problem-matcher-lint-spellintian.json"
+      - name: spellintian
+        run: ./scripts/spelling.sh spellintian
+  codespell:
+    name: codespell
+    runs-on: ubuntu-latest
+    container: debian:stable
+    needs: build
+    steps:
+      - name: Download built source tree archive
+        uses: actions/download-artifact@v3
+        with:
+          name: ola-debian-stable-built-source-tree
+          path: .
+      - name: SHA256 artifact archive
+        run: sha256sum ola-debian-stable-built-source-tree.tar.gz
+      - name: Unarchive artifacts and delete archive
+        shell: bash
+        run: |
+          tar -xvzf ola-debian-stable-built-source-tree.tar.gz .
+          rm ola-debian-stable-built-source-tree.tar.gz
+      - name: Display structure of extracted files
+        if: env.ACTIONS_STEP_DEBUG == 'true'
+        run: ls -alR
+      - name: Update package database
+        run: apt-get update -y
+      - name: Install lint tools
+        run: apt-get -y install python3-pip moreutils
+      - name: Install Python lint tools
+        run: python3 -m pip install --no-input codespell
```

Comment:
I normally go for the full bleeding-edge from git install on the CI, in the hope we'd generally see it there before users will (and partly because I've worked on codespell too, as a catch for overly keen dictionary entries).

Reply:
Fixed! Although it looks like I have some more exclusions to add.

Reply:
OK, I fixed most of the spelling issues. However I can't figure out how to get it to exclude the word `WRONLY`. I also tried `O_WRONLY` and `_WRONLY`.

Reply:
@peternewman let me know how I should best get codespell to ignore `WRONLY`.

Reply:
Fixed in another comment...